recon/core/playbook.py

```
@@ -0,0 +1,150 @@
+            results.append({"step": idx, "module": mod_name, "error": "module_not_found"})
+            continue
+        args = _format_args(step.get("args", {}), params)
+        args["http"] = http
+        # Let module defaults and validation handle missing optional args
+        try:
+            ok, miss = mod.validate(args)
+            if not ok and miss:
+                results.append({"step": idx, "module": mod_name, "error": "missing_required_options", "missing": miss})
+                continue
+        except Exception:
+            pass
```

Ruff (0.13.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)